### ISSUE TITLE
CLI not connecting to Daemon - macOS 10.13.5 - GUI does connect

### ISSUE THREAD

**Issue by apxs94** [monero-project/monero#4053]
Hi,

- macOS 10.13.5 on 2017 Macbook Pro

- Tested using 0.12.0.0 and 0.12.2.0

Error message I'm getting is:
```
Starting refresh...
Error: refresh failed: no connection to daemon. Please make sure daemon is running.. Blocks received: 0
Background refresh thread started
[wallet 494sdi (out of sync)]: 
```
- No antivirus or software firewall running.
- Blockchain is running from extermal USB
- GUI does sync from the daemon (oddly), just not CLI

Looking at the CLI log, this is the main error I get:

```
2018-06-25 19:26:42.620   0x7fff98967380        ERROR   net.http        contrib/epee/include/net/http_client.h:456      Unexpected recv fail
2018-06-25 19:26:42.622   0x7fff98967380        ERROR   wallet.wallet2  src/wallet/wallet2.cpp:1706     !r. THROW EXCEPTION: error::no_connection_to_daemon
2018-06-25 19:26:42.623   0x7fff98967380        WARN    net.http        src/wallet/wallet_errors.h:794  /DISTRIBUTION-BUILD/src/wallet/wallet2.cpp:1706:N5tools5error23no_connection_to_daemonE: no connection to daemon, request = gethashes.bin
2018-06-25 19:26:42.627   0x7fff98967380        ERROR   msgwriter       src/common/scoped_message_writer.h:102  Error: refresh failed: no connection to daemon. Please make sure daemon is running.. Blocks received: 0
```

Unsure how to further debug, help much appreciated.

**Comment by apxs94** [comment]
@stoffu 
Can you recommend any further debugging I could do for this?

**Comment by stoffu** [comment]
Probably it's because of the timeout being too low, which was fixed in #3962.

**Comment by apxs94** [comment]
@stoffu 
Thanks for the reply! Appreciate it.
Ok, will cross my fingers #3962 fixes it.
If I want to test, I can either compile latest master, or wait until it's released.
Just checking if they're my two options, as I'm not v familiar with github/builds etc.

**Comment by moneromooo-monero** [comment]
Fixed with current master/0.13 branch ?

**Comment by moneromooo-monero** [comment]
Thanks for checking.

+resolved
